src/lib/y2partitioner/actions/controllers/fstabs.rb

```diff
@@ -173,12 +185,30 @@ def can_be_imported?(entry)
           device = entry.device(system_graph)
           return false unless device
 
-          # Checks whether the device is actually a filesystem (i.e., NFS)
-          return true if device.is?(:filesystem)
+          return true unless must_be_formatted?(device, entry.mount_point)
 
           known_fs_type?(entry) && can_be_formatted?(device)
         end
 
+        def must_be_formatted?(device, mount_point)
+          return false unless format_system_volumes?
+
+          # Checks whether the device is actually a filesystem (i.e., NFS)
+          return false if device.is?(:filesystem)
+
+          system_volume_mount_point?(mount_point) || !device.formatted?
+        end
+
+        def system_volume_mount_point?(mount_point)
+          system_volume_mount_points.include?(mount_point)
+        end
+
+        def system_volumes_mount_points
+          mount_points = SYSTEM_VOLUMES_MOUNT_POINTS.dup
+          mount_points.delete("/boot/zipl") unless arch.s390?
```

Comment:
what about different approach and having arch specific array. which is concatenated for given architecture?